tests/rados/test_osd_ecpool_inconsistency_scenario.py

```diff
@@ -111,10 +124,12 @@ def run(ceph_cluster, **kw):
             f"The osd_scrub_auto_repair_num_errors value is set to {auto_repair_param_value}"
         )
 
-        mon_obj.set_config(section="osd", name="osd_scrub_auto_repair", value="true")
-        log.info("The osd_scrub_auto_repair value is set to true")
+        scrub_obj.set_osd_flags("set", "nodeep-scrub")
```

Comment:
Let's add this before we enable `osd_scrub_auto_repair` to true, just to avoid any race conditions

Reply:
Added the the nodeep-scrub before osd_scrub_auto_repair to true.